cassandra/cluster.py

```diff
@@ -3658,6 +3682,15 @@ def _try_connect(self, host):
                 "SCHEMA_CHANGE": partial(_watch_callback, self_weakref, '_handle_schema_change')
             }, register_timeout=self._timeout)
 
+            if self._cluster.check_tablets_enabled():
+                try:
+                    self._refresh_tablets(connection)
+                except InvalidRequest:
+                    self._cluster.experimental_tablet_feature_enabled = False
+                    self._cluster.load_balancing_policy.populate(self._cluster, self._cluster.metadata.all_hosts())
```

Comment:
I didn't see this line before, what does it do and we do we need it?

Reply:
We pass the information about tablets to load_balancing_policy during `populate` that happens before this check, so if there is an error during `_refresh_tablets` we need to change field in cluster, but also propagate this information to load_balancing_policy, without this line it will try to use tablets

Reply:
Makes sense. Could you put that in a comment?

Reply:
Done